.github/workflows/check-licenses.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           ref: ${{ github.head_ref }}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
```

Comment:
Will that work with the other triggers (workflow_dispatch, ~call, and push) above?

Reply:
I tested with the Push to an existed PR and works.
can't test for the moment with the call. However, this step will be only for the PR.

Reply:
As agreed, I will create a maintenance task to be tested after the integration to the main due to the limitation of testing.